src/assets/config_files/create_singlecellmodeling_structure.py

```diff
@@ -3,82 +3,104 @@
 Usage: create_singlecellmodeling_structure.py output.json
 '''
 
+import collections
 import json
-import requests
 import logging
 import os
+import requests
 import sys
 
-logging.basicConfig(level=logging.DEBUG)
-
 CSCS_OBJECT_STORAGE = os.environ['MODELS_URL']
 
 REQUIRED_FILES_NAME = {
     'morph': '_morph.jpeg',
     'responses': '_responses.png',
     'meta': 'meta.json',
 }
+OLD_LIST_NAME = 'old_model_list.txt'
 
 
-def _check_consistency(base_url, metadata, assets_list):
+def _check_consistency(base_url, metadata, required_names):
     for required in ('responses', 'morph'):
         asset = os.path.join(base_url, metadata[required])
-        if asset not in assets_list:
+        if asset not in required_names[required]:
             logging.warning('ERROR in cell. Plot not found %s', asset)
 
 
-def _check_models_modification(response):
-    old_list_name = 'old_model_list.txt'
-    if os.path.isfile(old_list_name):
-        x = open(old_list_name).read()
-        if x == response.text:
+def _check_models_modification(new_text):
+    new_text = list(sorted(new_text))
+    if os.path.isfile(OLD_LIST_NAME):
+        with open(OLD_LIST_NAME) as fd:
+            old_text = list(sorted(fd.read().split()))
+
+        if old_text == new_text:
             logging.debug('Models were not modified')
             return False
-        else:
-            logging.debug('Model list was changed')
 
-    # create temp file to compare next time
-    with open(old_list_name, 'w') as fd:
-        fd.write(response.text)
+    logging.debug('Model list was changed')
+
+    # create file to compare next time
+    with open(OLD_LIST_NAME, 'w') as fd:
+        fd.write('\n'.join(new_text))
+
     return True
 
 
+def filter_required_names(names, required):
+    ret = collections.defaultdict(set)
+    for name in names:
+        for k, ending in required.items():
+            if not name.endswith(ending):
+                continue
```

Comment:
One line is not used very much? like `if bla: continue`

Reply:
yeah, for some reason python style guides usually frown on single line statements, so I have it drilled in my head :)